src/client.ts

```diff
@@ -38,8 +38,14 @@ interface ClientEnqueueOptions {
   workflowName: string;
   /**
    * The name of the class containing the method that will be invoked when the workflow runs.
+   * If not provided, an empty string will be used as the class name.
    */
-  workflowClassName: string;
+  workflowClassName?: string;
```

Comment:
workflowClassName is only for configured instance registered workflows, so I made it optional